guide.md

```diff
@@ -11836,9 +11836,9 @@ But what if the service you want does not accept Monero but does accept a more m
 
 **Bitcoin and other "mainstream cryptocurrencies" are not anonymous at all (Remember [Your Cryptocurrencies transactions][Your Cryptocurrencies transactions:]) and you should never ever purchase, for example, Bitcoin from an exchange and then use these directly for purchasing services anonymously. This will not work, and the transaction can be traced easily.**
 
--   **Stay away from Crypto Mixers and Tumblers.** You might think this is a good idea but not only are they useless with cryptocurrencies such as BTC/ETH/LTC, but they are also dangerous as you might end up trading your currency for dirty currency from illicit activities. Use Monero to anonymize your crypto. Use a normal KYC-enabled Exchange to buy/sell your Monero (such as Kraken) or (at your own risk), use a service like LocalMonero.
+-   **Stay away from Crypto Mixer, Tumblers and Coinjoiners.** You might think this is a good idea but not only are they useless with cryptocurrencies such as BTC/ETH/LTC, but they are also dangerous as you might end up trading your currency for dirty currency from illicit activities. Use Monero to anonymize your crypto. Use a normal KYC-enabled Exchange to buy/sell your Monero (such as Kraken) or (at your own risk), use a service like LocalMonero.
 
--   **Stay away from what are in my opinion risky private/anonymizing wallets such as <https://we.incognito.org>.** Use a safer method outlined below.
+-   **Have a look below a couple pages below at "Warning about special tumbling, mixing, coinjoining privacy wallets and services" before proceeding.** 

```

Comment:
?

Reply:
The reference doesn't render properly. See https://github.com/Anon-Planet/thgtoa/blob/pr/162/guide.md#appendix-z-online-anonymous-payments-using-cryptocurrencies

Reply:
Table of Contents:
```suggestion
    -   [Warning about special tumbling, mixing, coinjoining privacy wallets and services] 
```
Line 11841:
```suggestion
-   **See [Warning about special tumbling, mixing, coinjoining privacy wallets and services].**
```

Line 13727:
```suggestion
  [Warning about special tumbling, mixing, coinjoining privacy wallets and services]: #warning-about-special-tumbling-mixing-coinjoining-privacy-wallets-and-services-wikiless-archiveorg 
```